src/ngMessages/messages.js

```diff
@@ -379,9 +398,17 @@ angular.module('ngMessages', [])
              messageCtrl.detach();
            });
 
-           unmatchedMessages.length !== totalMessages
-              ? $animate.setClass($element, ACTIVE_CLASS, INACTIVE_CLASS)
-              : $animate.setClass($element, INACTIVE_CLASS, ACTIVE_CLASS);
+           if (unmatchedMessages.length !== totalMessages) {
+             // Unset default message if setted
```

Comment:
`X  -->  Y` is my way of telling that there is a typo in `X`, it should be changed to `Y` :stuck_out_tongue: 
(Basically, change `setted` to `set`.)